source-code/telemetry/src/shared-events/coreUsedConfigEvent.ts

```diff
@@ -14,7 +14,7 @@ import type { InlangConfig } from "@inlang/core/config"
  *
  */
 export const coreUsedConfigEvent = {
-	name: "CORE used config" as const,
+	name: "Ecosystem used config" as const,
```

Comment:
FYI now you are breaking telemetry, seems worth it. 

- old insights must be updated
- ECOSYSTEM should be capitalized
- the event name should be changed to `ecosystemUsedConfigEvent`

Reply:
OK, I get it. We don't want to change a global function at some point; this is an open door for bugs. Right? 

I did it in this case because ECOSYSTEM is much more understandable than CORE.